### ISSUE TITLE
Color ramps

### ISSUE THREAD

**Issue by JohnTigue** [reconstrue/public_website#1]
Get color ramps going for Red, Green, Black of logo.

[Designing Systematic Colors](https://uxplanet.org/designing-systematic-colors-b5d2605b15c)
>How to make themable, flexible, WCAG 2.1 compliant color ramps for a design system

**Comment by JohnTigue** [comment]
Colors:
- Green:  #008000 00a800
- Red:    #aa0000
- Yellow: #f7b900
- Off white: #f8f8ff
- Off black: ??? ([#303030](https://www.colourlovers.com/color/303030/Not_Quite_Black) is too off)
